scripts/dev_setup.sh

```diff
@@ -11,6 +11,15 @@
 # 2 .profile will be used to configure the shell
 # 3 ${HOME}/bin/ is expected to be on the path - hashicorp tools/hadolint/etc.  will be installed there on linux systems.
 
+# fast fail.
+set -eo pipefail
```

Comment:
fast fail all errors in the script.   it's far more touchy now and all commands must return the success code.   mostly lead to ensuring grep didn't return error codes....